platform/windows/display_server_windows.cpp

```diff
@@ -6704,6 +6777,14 @@ Vector2i _get_device_ids(const String &p_device_name) {
 	return ids;
 }
 
+Vector2i _get_device_ids(const String &p_device_name) {
+	Vector2i out = _get_device_ids_info_reg(p_device_name);
+	if (out == Vector2i()) {
+		out = _get_device_ids_info_wmi(p_device_name);
```

Comment:
Is it possible for the registry not to have the ids while WMI does? Or is this fallback just there to be absolutely safe?

Reply:
Wine can have it missing (but WMI will fail on Wine as well), on real Windows it seems to have it, so fallback probably not needed.

Reply:
That's a good point. Let's go with this approach then.